addons/website_mass_mailing/static/src/scss/website_mass_mailing_popup.scss

```diff
@@ -0,0 +1,18 @@
+.o_newsletter_modal {
+    .o_popup_close {
+        @include o-position-absolute($right: -10px, $top: -10px);
+        cursor: pointer;
+        width: 30px;
+        height: 30px;
+        z-index: 1;
+        padding: 5px;
+        box-shadow: 1px 2px 2px 1px rgba(0, 0, 0, 0.2);
+    }
+    .o_mm_img_block {
+        background-position: center;
+        background-size: cover;
+    }
+}
+.o_newsletter_toast {
+    @include o-position-absolute($right: 15px, $top: 52px);
```

Comment:
So bootstrap toast have no default position in the page ? Maybe we should add this rule for all toasts in website.scss, with variables.